app/scripts/minmax/dimMinMax.controller.js

```diff
@@ -311,12 +315,10 @@
                         } else {
                             set_map[tiers_string] = [set];
                         }
-                        vm.allSetTiers = Object.keys(set_map).sort().reverse();
-                        vm.activesets = vm.allSetTiers[0];
                     }
                     
                     processed_count++;
-                    if((processed_count%200) == 0) {
+                    if((processed_count%1000) == 0) {
```

Comment:
Seems the lower the number in the modulo the longer it takes, but makes the page 'feel' more responsive. Commenting out that block my hunter finish in ~ 375ms +- 50ms, but essentially freezes the page.
%50000 ~ 375ms
%10000 ~ 375ms
%7500 ~ 375ms
%5000 ~ 400ms
%1000 ~ 500ms 
%200 ~ 1200ms

Seems to plateau for me around %7500. 

5000 seems like a good middle ground, let's go with that and if in the beta people are reporting issues, we can modify it. 

We can also get another minor speed boost from creating a outside/above the `step` function:

`var combos = (helms.length * gaunts.length * chests.length * legs.length * classItems.length * ghosts.length * artifacts.length) || 1;`

Using combos in place of doing that calc each time. Surprisingly takes my titan from 4s -> 2.5s and my hunter to 500ms. (tested at %1000.)